tests/libres_tests/integration/test_parameter_sample_types.py

```diff
@@ -146,6 +149,20 @@ def test_field_param(tmpdir, config_str, expected):
         # forward init was not set correctly
         assert load_from_forward_model(ert) == 1
 
+        fs = ert.getEnkfFsManager().getFileSystem("default_0", read_only=True)
+        if expected:
+            arr = fs.load_parameters(
+                ert.ensembleConfig(), [0], [update.Parameter("MY_PARAM")]
+            )
+            assert len(arr) == 16
+        else:
+            # load_parameters should probably handle errors better than
```

Comment:
Create an issue instead of comment?

Reply:
I created this issue https://github.com/equinor/ert/issues/3871. Perhaps I could add the issue to the comment

Reply:
Personally not a big fan of issues in comments since they tend to stick around.